app/controllers/application_controller.rb

```diff
@@ -1891,6 +1891,8 @@ def controller_for_common_methods
       "generic_object"
     when "ansible_playbook"
       "embedded_configuration_script_payload"
+    when "workflow_repository"
+      "ansible_repository"
```

Comment:
This is a little weird.  Since this is in `controller_for_common_methods`, is this going to the other controller?

Reply:
No, this is just used by the function `tagging_edit(db = nil, assert = true)` to say that the workflow repository has the same permissions as the ansible repository. It is still running the workflow repository controller code.